app/controllers/hosts_controller.rb

```diff
@@ -312,7 +312,9 @@ def nics
   def forget_status
     status = @host.host_statuses.find(params[:status])
     status.delete
-    redirect_to host_path(@host)
+    head :ok
+  rescue => exception
+    process_ajax_error exception, 'forget status'
```

Comment:
This is not stated in `AJAX_REQUESTS`, and it was proper action before this change, now it's AJAX only.
Shouldn't we have case for `request.xhr?` I'd expect this is used somewhere else as well.

Reply:
The existing host detail page probably uses non-ajax version. As @ezr-ondrej mentions, we can have a single controller action with different behavior based on `request.xhr?` or render different response.